pkg/util/utils.go

```diff
@@ -669,3 +681,7 @@ func swapSELinuxLabel(cLabel, processLabel string) (string, error) {
 	dcon["type"] = scon["type"]
 	return dcon.Get(), nil
 }
+
+func DefaultContainerConfig() *config.Config {
```

Comment:
Should cmd/podman/registry/config.go be updated to use this rather than calling config directly?